packages/lib/models/Setting.ts

```diff
@@ -146,6 +146,7 @@ export interface Constants {
 	pluginDataDir: string;
 	cacheDir: string;
 	pluginDir: string;
+	homeDir: string;
```

Comment:
I'm not sure that `homeDir` is the best name for this setting...

Currently, `homeDir` points to `os.homedir()` only when not using the portable application (or the `--profile` flag).